### DISCUSSION TITLE
About rigid body motion (RBM) in two particle sintering simulation

### DISCUSSION THREAD

**Discussion by Clark-domain** [idaholab/moose#18305] [Q&A Modules: Phase field] [answered]
Hi, everyone,

I got the question about using rigid body motion for two-particle sintering with phase field module.

I tried the input file from (modules/phase_field/test/tests/rigidbodymotion/polycrystal_action.i) which worked. I found that when I set up the two-particle along x-direction as an initial condition, the simulation itself ran well but I could not see the reaction force/advection velocity from Paraview. 

Since the physics for two-particle sintering should not rely on two-particle orientation for isotropic diffusion, I wanted to know why I could not see the reaction force/advection velocity. Is it ok to use it for two-particle along x-direction? Do I need to change rotation_constant/translation_constant/stiffness_constant?

Thanks very much,

Kaijie

**Comment by SudiptaBiswas** [comment]
What do you mean by 'could not see the reaction force'? Did the simulation not output the forces for visualization or were the values zero? 
First, please make sure you are properly outputting the variables you are interested in. Next, when two particles are aligned along the same axes, there will not be any rotation. There could be some translation though depending on the force values. You can find a bit more details here, https://link.springer.com/article/10.1007/s10853-017-1846-3.

**Reply by Clark-domain** [reply]
Thanks very much.

I should say the outputting variables were all zero for my case when I put particles along the same x-axis. I have been reading that particle and other related articles many times, which were great work. And I wanted to apply this PFM model to my studies.

Here are the pictures for you to see the test case from MOOSE and the application to my case. Is there a problem with my case? But your paper showed the different Vx/Vy to mine. I do not why?

Test case for c and force df0:
![test1](https://user-images.githubusercontent.com/74393696/126504749-07e99f31-5196-459e-a81d-3ac818729935.png)

Test case for velocity x and y:
![test2](https://user-images.githubusercontent.com/74393696/126504793-b2cd80a0-7df7-4768-bbe9-54330232cac2.png)

My case for c and force df0:
![me1](https://user-images.githubusercontent.com/74393696/126504821-99af7006-7eac-475b-8700-defbe60e9103.png)

My case for velocity x and y:
![me2](https://user-images.githubusercontent.com/74393696/126504835-0bad5344-7877-4dcf-ac05-f38b7a4a8c1d.png)

**Comment by SudiptaBiswas** [comment, marked as answer]
For the first case, you are looking at the total advection velocities whereas the paper shows the translation and rotation components separately. That's why the difference. 

For the second case, it might be that the particles don't have enough overlap to generate the forces. Try increasing the interface width and look at a later timestep. Although,  it's hard to pinpoint the reason just from the image.

**Reply by Clark-domain** [reply]
You are correct. I just finished my new test, and it worked as seen below. Thanks very much.


![image](https://user-images.githubusercontent.com/74393696/127774642-94cf233c-2914-4def-8265-c93aa1df4bdc.png)
